gloss/crt0.S

```diff
@@ -11,6 +11,8 @@
 
 /* crt0.S: Entry point for RISC-V METAL programs. */
 
+#include <metal/machine/platform.h>
```

Comment:
I am not sure it is a good to include this.
Up to now, crt0.S is only depend the linker file I believe